Cartfile.private

```diff
@@ -1,2 +1,2 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.4
-github "AliSoftware/OHHTTPStubs" ~> 6.0
+github "linksmt/OHHTTPStubs" "563f48d3fab84ef04639649c770b00f4fa502cca"
```

Comment:
Is there a PR for this change or something else we can keep track of for when master gets updated? AliSoftware/OHHTTPStubs#295 is already in [v7.0.0](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/7.0.0); is this for something else?

Reply:
Oh, looks like this commit also pulls in AliSoftware/OHHTTPStubs#286. The CircleCI configuration isn’t testing on watchOS anyways. So I think we can pin to the main pod’s v7._x_.

Reply:
Yea, the reason why I switched to the fork here is the missing watchOS scheme in the original. We don't test on watchOS but we do compile for watchOS. We could switch back but it would require a workaround to run carthage bootstrap and ignore the .private and .resolved files.